src/import-data/ImportDataDestination.ts

```diff
@@ -11,19 +19,40 @@ import colors from 'src/libs/colors';
 import { FormLabel } from 'src/libs/forms';
 import * as Style from 'src/libs/style';
 import * as Utils from 'src/libs/utils';
+import { WorkspaceInfo } from 'src/libs/workspace-utils';
 
+import { TemplateWorkspaceInfo } from './import-types';
 import { isProtectedWorkspace } from './protected-data-utils';
 
 const styles = {
+  card: {
+    borderRadius: 5,
+    backgroundColor: 'white',
+    padding: '2rem',
+    flex: 1,
+    minWidth: 0,
+    boxShadow: Style.standardShadow,
+  },
```

Comment:
I'm confused where this is coming from. It looks like it duplicates `styles.card` from ImportDataOverView. I guess the code was using `styles.card` before but it never defined it?

Reply:
Correct. Converting to TS revealed a bug from the initial breakup of ImportData.js into multiple modules (#4213). This component used this style, but I missed copying it into this module.